typed-racket-doc/typed-racket/scribblings/reference/special-forms.scrbl

```diff
@@ -408,11 +411,13 @@ amount it needs.
 ]
 
 Options provided have the same meaning as for the @|struct-id| form
-from @racketmodname[racket/base].
+from @racketmodname[racket/base] (with the exception of @racket[#:type-name], as
+described above).
 
-A prefab structure type declaration will bind the given @racket[name] to a
-@racket[Prefab] type. Unlike in @racketmodname[racket/base], a non-prefab
-structure type cannot extend a prefab structure type.
+A prefab structure type declaration will bind the given @racket[name-id]
+or @racket[type-id] to a @racket[Prefab] type. Unlike in
+@racketmodname[racket/base], a non-prefab structure type cannot extend
```

Comment:
I think this should say something like "unlike the `struct` form from `racket/base`" with the appropriate link.